### PR TITLE
👷 📦 Postgres tests for Jenkins

### PR DESCRIPTION
**Pull request by ElRaptorus**
**Changes:**

1. Add parallel build Step to Jenkins file that runs the tests against PostgreSQL
2. Update the Jenkinsfile to the new design specs employed by the ProcessEngine Runtime
3. Delete obsolete docker testfile
4. Only send a full testlog to slack, if at least one test has failed

PR: #192

## How can others test the changes?

Watch a build for the Consumer API. 
For example:
https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fprocess_engine_meta/detail/feature%2Fadd_postgres_tests_for_jenkins/3/pipeline

You'll see that the tests are now performed against Sqlite and PostgreSQL.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -60,73 +60,167 @@ def slack_send_testlog(testlog) {
...
+          agent any
+          options {
+            skipDefaultCheckout()
+          }
+          steps {
+            dir('_integration_tests') {
+              unstash('post_build');
+
+              script {
+
```
Comment (rrrene) [inline Jenkinsfile]:
```suggestion
```

File: Jenkinsfile [hunk 2]
```diff
@@ -60,73 +60,167 @@ def slack_send_testlog(testlog) {
...
+            label 'macos'
+          }
+          options {
+            skipDefaultCheckout()
+          }
+          steps {
+            dir('_integration_tests') {
+              unstash('post_build');
+
+              script {
```
Comment (rrrene) [inline Jenkinsfile]:
```suggestion
```

File: Jenkinsfile [hunk 3]
```diff
@@ -60,73 +60,167 @@ def slack_send_testlog(testlog) {
...
-            currentBuild.result = 'SUCCESS'
-            if (error_code > 0) {
-              test_failed = true;
-              currentBuild.result = 'FAILURE'
+                  docker.image('postgres:11').inside("--link ${c.id}:${postgres_host}") {
+                    sh "while ! pg_isready --host ${postgres_host} --username ${postgres_username} --dbname ${postgres_database}; do sleep 1; done"
+                  };
+
+                  docker.image("node:${NODE_VERSION_NUMBER}").inside("--link ${c.id}:${postgres_host} --env PATH=$PATH:/$WORKSPACE/node_modules/.bin") {
+
```
Comment (rrrene) [inline Jenkinsfile]:
```suggestion
```

**Comment by ElRaptorus** [comment]
@rrrene done

**Pull request merged**